### DISCUSSION TITLE
Add gif support 🖼

### DISCUSSION THREAD

**Discussion by Bad3r** [community/community#9068] [Mobile]
Currently it's not possible to view gifs in the iOS mobile client. 
![image](https://user-images.githubusercontent.com/25513724/146657497-56ec0693-8989-4c1a-a997-bc5bf31b7b13.jpeg)

**Comment by mxie** [comment]
Thanks for sharing your feedback @Bad3r! We've relayed this back to our team to assess. Stay tuned for updates!

**Reply by Bad3r** [reply]
@mxie its been a year. Any updates?
